consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseElectionAPI.java

```diff
@@ -566,6 +570,12 @@ private void setGeneralFields(Election election, String referenceId, ElectionTyp
         if (StringUtils.isEmpty(election.getStatus())) {
             election.setStatus(ElectionStatus.OPEN.getValue());
         }
+        setVersion(election, referenceId);
+    }
+
+    private void setVersion(Election election, String referenceId) {
+        Integer version = electionDAO.getLastVersionByReferenceId(referenceId);
+        election.setVersion(version == null ? 1 : ++version);
```

Comment:
I'm a little concerned about this being here. I think it would be safer to calculate this value as part of the insert statement itself. Having this in code makes it easier to calculate, but subject to subtle race condition bugs.